scripts/dev/contexts/e2e_smoke_release

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -Eeou pipefail
+
+script_name=$(readlink -f "${BASH_SOURCE[0]}")
+script_dir=$(dirname "${script_name}")
+
+source "${script_dir}/root-context"
+source "${script_dir}/variables/om70"
+
+export DATABASE_REGISTRY="${QUAY_REGISTRY}"
+export APPDB_REGISTRY="${QUAY_REGISTRY}"
+export INIT_OPS_MANAGER_REGISTRY="${QUAY_REGISTRY}"
+export OPS_MANAGER_REGISTRY="${QUAY_REGISTRY}"
+export OPERATOR_REGISTRY="${QUAY_REGISTRY}"
+export INIT_IMAGES_REGISTRY="${QUAY_REGISTRY}"
+export INIT_APPDB_REGISTRY="${QUAY_REGISTRY}"
+export INIT_DATABASE_REGISTRY="${QUAY_REGISTRY}"
+# Since we're sourcing this as an initial step, the jq might not be there. That's why we need bash magic here.
+OPERATOR_VERSION="$(grep -o '"mongodbOperator": "[^"]*' release.json | grep -o '[^"]*$')"
+export OPERATOR_VERSION
+INIT_DATABASE_VERSION="$(grep -o '"initDatabaseVersion": "[^"]*' release.json | grep -o '[^"]*$')"
+export INIT_DATABASE_VERSION
+INIT_APPDB_VERSION="$(grep -o '"initAppDbVersion": "[^"]*' release.json | grep -o '[^"]*$')"
+export INIT_APPDB_VERSION
+INIT_OPS_MANAGER_VERSION="$(grep -o '"initOpsManagerVersion": "[^"]*' release.json | grep -o '[^"]*$')"
+export INIT_OPS_MANAGER_VERSION
+DATABASE_VERSION="$(grep -o '"databaseImageVersion": "[^"]*' release.json | grep -o '[^"]*$')"
+export DATABASE_VERSION
```

Comment:
nit: is this not possible? 🤔 

```suggestion
export DATABASE_VERSION="$(grep -o '"databaseImageVersion": "[^"]*' release.json | grep -o '[^"]*$')"
```

Reply:
Probably it's possible, let's see ;) I just copied the existing `e2e_smoke` and `e2e_static_smoke`, didn't dive into what's exactly inside

Reply:
<img width="880" alt="Screenshot 2025-05-13 at 10 12 34" src="https://github.com/user-attachments/assets/ba7ddfa2-f67c-4572-9269-4d3057d54b70" />